status/format.go

```diff
@@ -249,3 +261,17 @@ func FormatTaskLogURL(taskRun *helpers.TaskRun, pipelinerun string, namespace st
 	}
 	return buf.String()
 }
+
+func EvaluateReason(taskRun *helpers.TaskRun) (string, error) {
```

Comment:
I may be missing something, but I'm not seeing where this function is used?

Reply:
You are not missing anything, actually I missed this one. Interesting that CI didn't catch that.